src/main/client/scene/main-menu-scene.ts

```diff
@@ -0,0 +1,27 @@
+import { MenuButton } from "../ui/menu-button";
+
+export class MainMenuScene extends Phaser.Scene {
+   constructor() {
+      super({
+         active: false,
+         visible: false,
+         key: "MainMenu" // TODO: Extract key
+      });
+   }
+
+   create(): void {
+      this.add
+         .text(100, 50, "This is a sample main menu. Click the \"Start\" button below to run your game.", {
+            fill: "#FFFFFF"
+         })
+         .setFontSize(24);
+
+      new MenuButton(this, 100, 150, "Start Game", () => {
```

Comment:
not sure if i like this, 

it creates a new button, then doesn't use the instantiated object

Reply:
Sure, I just copied it from an example repo, feel free to change the ui.

Reply:
nice